cfme/common/vm.py

```diff
@@ -522,20 +521,19 @@ class VM(BaseVM):
     TO_RETIRE = None
 
     retire_form = Form(fields=[
-        ('date_retire', {
-            version.LOWEST: date_retire_element,
-            "5.5": AngularCalendarInput(
-                "retirement_date", "//label[contains(normalize-space(.), 'Retirement Date')]")}),
+        ('date_retire', AngularCalendarInput(
+            "retirement_date", "//label[contains(normalize-space(.), 'Retirement')]")),
```

Comment:
But what if new element will be added before calendar widget with similar locator? It will start to interact with it  and can redirect you to another page/menu, for example